setup.py

```diff
@@ -64,14 +53,16 @@
             'openfisca-run-test=openfisca_core.scripts.openfisca_command:main',
             ],
         },
+    python_requires = ">= 3.7",
+    install_requires = open("requirements/install").readlines(),
```

Comment:
Why not use the same loading mechanism (`require` / `load_requirements_from_file`) as the `extras_require`?

Reply:
Leftover! Thanks :)